module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderContentCard.kt

```diff
@@ -0,0 +1,31 @@
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.Modifier
+import com.github.ajalt.colormath.extensions.android.composecolor.toComposeColor
+import org.cru.godtools.shared.renderer.state.State
+import org.cru.godtools.shared.tool.parser.model.Card
+
+@Composable
+internal fun ColumnScope.RenderContentCard(
+    card: Card,
+    state: State,
+    modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(10.dp)
+) {
+    Column(
```

Comment:
there is a `Card` composable in material3 you can use for rendering the card UI itself